src/build/arg/mod.rs

```diff
@@ -4116,55 +4113,7 @@ impl<'help> Arg<'help> {
         self.multiple_occurrences(multi).multiple_values(multi)
     }
 
-    /// Allows an argument to accept explicitly empty values. An empty value must be specified at
```

Comment:
Please keep this documentation. We need to add `allow_empty_values` function.

Reply:
That's an accident, reverted.